resources/js/pages/home/Components/HomeSideProfile.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { MdOutlinePhotoLibrary } from "react-icons/md";
+import { FaRegCircleUser } from "react-icons/fa6";
+import { TiEdit } from "react-icons/ti";
+import { FiArchive } from "react-icons/fi";
+
+
+
+export default function HomeSideProfile () {
+    return (
+        <>
+        <div className='border border-slate-100 py-8 px-12 text-center'>
+            <img src="https://picsum.photos/101/101" className='rounded-full mb-5 mx-auto' />
```

Comment:
don't use fake data in your components or models, place for your fake data is on your json files
here you could using your User model get one of your models and use it here